.tekton/verify-enterprise-contract-task-main-ci-pull-request.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && files.modified.exists(x, x.matches("verify-enterprise-contract.yaml"))
```

Comment:
```suggestion
    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && files.modified.exists(x, x.endsWith("verify-enterprise-contract.yaml"))
```

Reply:
Or... 
```suggestion
    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && files.modified.exists(x, x == "tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml"))
```

Reply:
Not sure if we would get `./tasks/...` or `/tasks/...` here, but I guess `tasks/...` seems most likely.